typing/untypeast.mli

```diff
@@ -83,3 +83,6 @@ val untype_structure : ?mapper:mapper -> Typedtree.structure -> structure
 val untype_signature : ?mapper:mapper -> Typedtree.signature -> signature
 
 val constant : Asttypes.constant -> Parsetree.constant
+
+val untype_expression : ?mapper:mapper -> Typedtree.expression -> expression
+val untype_pattern : ?mapper:mapper -> _ Typedtree.general_pattern -> pattern
```

Comment:
Nit: I would find it slightly more natural to see those functions listed right after the two other `untype_` and before `constant`.